apple/internal/partials/debug_symbols.bzl

```diff
@@ -153,7 +153,7 @@ def _copy_dsym_into_declared_bundle(
         actions.run_shell(
             inputs = [dsym_binary],
             outputs = [output_binary],
-            mnemonic = "DsymDwarf",
+            mnemonic = "CopyDWARF2dSYM",
```

Comment:
This is a breaking change, do we want to take it?

Reply:
I don't feel strongly. It seemed like a pretty minor one so I wasn't worried about it.

Reply:
I know there are people excluding this mnemonic, so it would be annoying if it just started being unexcluded. I vote not to take it.

Reply:
Yea, it's probably not worth breaking people. But in general we should be able to make changes like this as long as we mention them in the notes. In this example I think the upstream mnemonic is more descriptive, which is nicer.

Reply:
We reference `DsymDwarf` right now so for us this would be a breaking change. As long as it's clearly noted in the release notes, I'm fine with it.